pkg/kubelet/rkt/config.go

```diff
@@ -44,13 +53,54 @@ func (c *Config) buildGlobalOptions() []string {
 		return result
 	}
 
-	result = append(result, fmt.Sprintf("--debug=%v", c.Debug))
-	result = append(result, fmt.Sprintf("--insecure-options=%s", c.InsecureOptions))
+	if c.Debug {
+		result = append(result, "--debug=true")
+	}
+	if c.InsecureOptions != "" {
+		result = append(result, fmt.Sprintf("--insecure-options=%s", c.InsecureOptions))
+	}
 	if c.LocalConfigDir != "" {
 		result = append(result, fmt.Sprintf("--local-config=%s", c.LocalConfigDir))
 	}
+	if c.UserConfigDir != "" {
+		result = append(result, fmt.Sprintf("--user-config=%s", c.UserConfigDir))
+	}
+	if c.SystemConfigDir != "" {
+		result = append(result, fmt.Sprintf("--system-config=%s", c.SystemConfigDir))
+	}
 	if c.Dir != "" {
 		result = append(result, fmt.Sprintf("--dir=%s", c.Dir))
 	}
 	return result
 }
+
+// getConfig gets configurations from the rkt API service
+// and merge it with the existing config. The merge rule is
+// that the fields in the provided config will override the
+// result that get from the rkt api service.
+func (r *Runtime) getConfig(cfg *Config) (*Config, error) {
+	resp, err := r.apisvc.GetInfo(context.Background(), &rktapi.GetInfoRequest{})
+	if err != nil {
+		return nil, err
+	}
+
+	flags := resp.Info.GlobalFlags
```

Comment:
@luxas This is probably because the cached pkg is outdated. Could you try `make clean` before build?